code/modules/wiremod/components/hud/object_overlay.dm

```diff
@@ -0,0 +1,132 @@
+/**
+ * # Object Overlay Component
+ *
+ * Shows an overlay ontop of an object. Toggleable.
+ * Requires a BCI shell.
+ */
+
+GLOBAL_VAR_INIT(object_overlay_id, 0) //I need every object_overlay component to have a DIFFERENT ID because else they are going to overlap
```

Comment:
This needs a doc comment as two places used it, and should be moved up since it is used outside of object_overlays.